<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="11" ignored="0" total="25" passed="14">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="92516" started-at="2021-04-19T10:48:01 IST" finished-at="2021-04-19T10:49:34 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="92516" started-at="2021-04-19T10:48:01 IST" finished-at="2021-04-19T10:49:34 IST">
      <class name="Assignment.ReqResTesting.GetListOfUsersTest">
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.GetListOfUsersTest@731a74c]" name="precondition" is-config="true" duration-ms="11" started-at="2021-04-19T10:48:44 IST" finished-at="2021-04-19T10:48:44 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="checkListOfusers()[pri:0, instance:Assignment.ReqResTesting.GetListOfUsersTest@731a74c]" name="checkListOfusers" duration-ms="7255" started-at="2021-04-19T10:48:44 IST" finished-at="2021-04-19T10:48:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkListOfusers -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.GetListOfUsersTest@731a74c]" name="precondition" is-config="true" duration-ms="5" started-at="2021-04-19T10:48:51 IST" finished-at="2021-04-19T10:48:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="FAIL" signature="checkListOfusersWithoutQueryParam()[pri:0, instance:Assignment.ReqResTesting.GetListOfUsersTest@731a74c]" name="checkListOfusersWithoutQueryParam" duration-ms="5745" started-at="2021-04-19T10:48:51 IST" finished-at="2021-04-19T10:48:57 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [12] but found [6]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [12] but found [6]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at Assignment.ReqResTesting.GetListOfUsersTest.checkListOfusersWithoutQueryParam(GetListOfUsersTest.java:87)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkListOfusersWithoutQueryParam -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.GetListOfUsersTest@731a74c]" name="precondition" is-config="true" duration-ms="5" started-at="2021-04-19T10:48:57 IST" finished-at="2021-04-19T10:48:57 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="FAIL" signature="checkListOfuserswithInvalidPage()[pri:0, instance:Assignment.ReqResTesting.GetListOfUsersTest@731a74c]" name="checkListOfuserswithInvalidPage" duration-ms="2656" started-at="2021-04-19T10:48:57 IST" finished-at="2021-04-19T10:49:00 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [200]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [200]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at Assignment.ReqResTesting.GetListOfUsersTest.checkListOfuserswithInvalidPage(GetListOfUsersTest.java:67)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkListOfuserswithInvalidPage -->
      </class> <!-- Assignment.ReqResTesting.GetListOfUsersTest -->
      <class name="Assignment.ReqResTesting.CreateUserTest">
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="precondition" is-config="true" duration-ms="6011" started-at="2021-04-19T10:48:01 IST" finished-at="2021-04-19T10:48:07 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="FAIL" signature="checkUserCreation(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="checkUserCreation" duration-ms="7040" started-at="2021-04-19T10:48:07 IST" data-provider="getUserCreationData" finished-at="2021-04-19T10:48:14 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [201]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [201]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at Assignment.ReqResTesting.CreateUserTest.checkUserCreation(CreateUserTest.java:44)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkUserCreation -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="precondition" is-config="true" duration-ms="5" started-at="2021-04-19T10:48:14 IST" finished-at="2021-04-19T10:48:14 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="checkUserCreation(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="checkUserCreation" duration-ms="4370" started-at="2021-04-19T10:48:14 IST" data-provider="getUserCreationData" finished-at="2021-04-19T10:48:19 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dummyjob]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkUserCreation -->
        <test-method status="PASS" signature="checkUserCreation(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="checkUserCreation" duration-ms="3496" started-at="2021-04-19T10:48:19 IST" data-provider="getUserCreationData" finished-at="2021-04-19T10:48:22 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[dummyuser]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkUserCreation -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="precondition" is-config="true" duration-ms="0" started-at="2021-04-19T10:48:19 IST" finished-at="2021-04-19T10:48:19 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="precondition" is-config="true" duration-ms="10" started-at="2021-04-19T10:48:22 IST" finished-at="2021-04-19T10:48:22 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="checkUserCreation(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="checkUserCreation" duration-ms="3960" started-at="2021-04-19T10:48:22 IST" data-provider="getUserCreationData" finished-at="2021-04-19T10:48:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[dummyuser]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dummyjob]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkUserCreation -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="precondition" is-config="true" duration-ms="0" started-at="2021-04-19T10:48:26 IST" finished-at="2021-04-19T10:48:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="FAIL" signature="checkUserCreationWithWrongBody(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="checkUserCreationWithWrongBody" duration-ms="3434" started-at="2021-04-19T10:48:26 IST" data-provider="getUserCreationData" finished-at="2021-04-19T10:48:30 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [201]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [201]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at Assignment.ReqResTesting.CreateUserTest.checkUserCreationWithWrongBody(CreateUserTest.java:70)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkUserCreationWithWrongBody -->
        <test-method status="FAIL" signature="checkUserCreationWithWrongBody(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="checkUserCreationWithWrongBody" duration-ms="2905" started-at="2021-04-19T10:48:30 IST" data-provider="getUserCreationData" finished-at="2021-04-19T10:48:33 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dummyjob]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [201]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [201]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at Assignment.ReqResTesting.CreateUserTest.checkUserCreationWithWrongBody(CreateUserTest.java:70)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkUserCreationWithWrongBody -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="precondition" is-config="true" duration-ms="0" started-at="2021-04-19T10:48:30 IST" finished-at="2021-04-19T10:48:30 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="precondition" is-config="true" duration-ms="5" started-at="2021-04-19T10:48:33 IST" finished-at="2021-04-19T10:48:33 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="FAIL" signature="checkUserCreationWithWrongBody(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="checkUserCreationWithWrongBody" duration-ms="2779" started-at="2021-04-19T10:48:33 IST" data-provider="getUserCreationData" finished-at="2021-04-19T10:48:35 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[dummyuser]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [201]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [201]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at Assignment.ReqResTesting.CreateUserTest.checkUserCreationWithWrongBody(CreateUserTest.java:70)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkUserCreationWithWrongBody -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="precondition" is-config="true" duration-ms="16" started-at="2021-04-19T10:48:35 IST" finished-at="2021-04-19T10:48:35 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="FAIL" signature="checkUserCreationWithWrongBody(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.CreateUserTest@5a8e6209]" name="checkUserCreationWithWrongBody" duration-ms="2780" started-at="2021-04-19T10:48:35 IST" data-provider="getUserCreationData" finished-at="2021-04-19T10:48:38 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[dummyuser]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dummyjob]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [201]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [201]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at Assignment.ReqResTesting.CreateUserTest.checkUserCreationWithWrongBody(CreateUserTest.java:70)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkUserCreationWithWrongBody -->
      </class> <!-- Assignment.ReqResTesting.CreateUserTest -->
      <class name="Assignment.ReqResTesting.LoginTest">
        <test-method status="PASS" signature="loginCheckForRegisteredUser(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.LoginTest@369f73a2]" name="loginCheckForRegisteredUser" duration-ms="2859" started-at="2021-04-19T10:49:00 IST" data-provider="getUserData" finished-at="2021-04-19T10:49:02 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginCheckForRegisteredUser -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.LoginTest@369f73a2]" name="precondition" is-config="true" duration-ms="0" started-at="2021-04-19T10:49:00 IST" finished-at="2021-04-19T10:49:00 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="loginCheckForRegisteredUser(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.LoginTest@369f73a2]" name="loginCheckForRegisteredUser" duration-ms="2970" started-at="2021-04-19T10:49:02 IST" data-provider="getUserData" finished-at="2021-04-19T10:49:05 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[pistol]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginCheckForRegisteredUser -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.LoginTest@369f73a2]" name="precondition" is-config="true" duration-ms="0" started-at="2021-04-19T10:49:02 IST" finished-at="2021-04-19T10:49:02 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.LoginTest@369f73a2]" name="precondition" is-config="true" duration-ms="6" started-at="2021-04-19T10:49:05 IST" finished-at="2021-04-19T10:49:05 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="loginCheckForRegisteredUser(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.LoginTest@369f73a2]" name="loginCheckForRegisteredUser" duration-ms="2820" started-at="2021-04-19T10:49:05 IST" data-provider="getUserData" finished-at="2021-04-19T10:49:08 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[eve.holt@reqres.in]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginCheckForRegisteredUser -->
        <test-method status="PASS" signature="loginCheckForRegisteredUser(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.LoginTest@369f73a2]" name="loginCheckForRegisteredUser" duration-ms="2819" started-at="2021-04-19T10:49:08 IST" data-provider="getUserData" finished-at="2021-04-19T10:49:11 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[eve.holt@reqres.in]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[pistol]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginCheckForRegisteredUser -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.LoginTest@369f73a2]" name="precondition" is-config="true" duration-ms="0" started-at="2021-04-19T10:49:08 IST" finished-at="2021-04-19T10:49:08 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.LoginTest@369f73a2]" name="precondition" is-config="true" duration-ms="5" started-at="2021-04-19T10:49:11 IST" finished-at="2021-04-19T10:49:11 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="loginCheckForUnRegisteredUser()[pri:0, instance:Assignment.ReqResTesting.LoginTest@369f73a2]" name="loginCheckForUnRegisteredUser" duration-ms="2844" started-at="2021-04-19T10:49:11 IST" finished-at="2021-04-19T10:49:14 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginCheckForUnRegisteredUser -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.LoginTest@369f73a2]" name="precondition" is-config="true" duration-ms="0" started-at="2021-04-19T10:49:14 IST" finished-at="2021-04-19T10:49:14 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="FAIL" signature="loginCheckWithWrongHttpMethod()[pri:0, instance:Assignment.ReqResTesting.LoginTest@369f73a2]" name="loginCheckWithWrongHttpMethod" duration-ms="2457" started-at="2021-04-19T10:49:14 IST" finished-at="2021-04-19T10:49:16 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [405] but found [200]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [405] but found [200]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at Assignment.ReqResTesting.LoginTest.loginCheckWithWrongHttpMethod(LoginTest.java:86)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginCheckWithWrongHttpMethod -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.LoginTest@369f73a2]" name="precondition" is-config="true" duration-ms="5" started-at="2021-04-19T10:49:16 IST" finished-at="2021-04-19T10:49:16 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="FAIL" signature="loginCheckWithWrongPassword()[pri:0, instance:Assignment.ReqResTesting.LoginTest@369f73a2]" name="loginCheckWithWrongPassword" duration-ms="3724" started-at="2021-04-19T10:49:16 IST" finished-at="2021-04-19T10:49:20 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [401] but found [200]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [401] but found [200]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at Assignment.ReqResTesting.LoginTest.loginCheckWithWrongPassword(LoginTest.java:73)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginCheckWithWrongPassword -->
      </class> <!-- Assignment.ReqResTesting.LoginTest -->
      <class name="Assignment.ReqResTesting.DeleteUserTest">
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.DeleteUserTest@4b4523f8]" name="precondition" is-config="true" duration-ms="5" started-at="2021-04-19T10:48:38 IST" finished-at="2021-04-19T10:48:38 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="deleteUser()[pri:0, instance:Assignment.ReqResTesting.DeleteUserTest@4b4523f8]" name="deleteUser" duration-ms="2817" started-at="2021-04-19T10:48:38 IST" finished-at="2021-04-19T10:48:41 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteUser -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.DeleteUserTest@4b4523f8]" name="precondition" is-config="true" duration-ms="3" started-at="2021-04-19T10:48:41 IST" finished-at="2021-04-19T10:48:41 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="FAIL" signature="deleteUserWithWrongId()[pri:0, instance:Assignment.ReqResTesting.DeleteUserTest@4b4523f8]" name="deleteUserWithWrongId" duration-ms="2785" started-at="2021-04-19T10:48:41 IST" finished-at="2021-04-19T10:48:44 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [204]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [204]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at Assignment.ReqResTesting.DeleteUserTest.deleteUserWithWrongId(DeleteUserTest.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteUserWithWrongId -->
      </class> <!-- Assignment.ReqResTesting.DeleteUserTest -->
      <class name="Assignment.ReqResTesting.RegisterTest">
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.RegisterTest@1f28c152]" name="precondition" is-config="true" duration-ms="5" started-at="2021-04-19T10:49:20 IST" finished-at="2021-04-19T10:49:20 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="FAIL" signature="loginCheckWithWrongHttpMethod()[pri:0, instance:Assignment.ReqResTesting.RegisterTest@1f28c152]" name="loginCheckWithWrongHttpMethod" duration-ms="2275" started-at="2021-04-19T10:49:20 IST" finished-at="2021-04-19T10:49:22 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [405] but found [200]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [405] but found [200]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at Assignment.ReqResTesting.RegisterTest.loginCheckWithWrongHttpMethod(RegisterTest.java:69)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginCheckWithWrongHttpMethod -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.RegisterTest@1f28c152]" name="precondition" is-config="true" duration-ms="15" started-at="2021-04-19T10:49:22 IST" finished-at="2021-04-19T10:49:22 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="m1(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.RegisterTest@1f28c152]" name="m1" duration-ms="2815" started-at="2021-04-19T10:49:22 IST" data-provider="getUserData" finished-at="2021-04-19T10:49:25 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- m1 -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.RegisterTest@1f28c152]" name="precondition" is-config="true" duration-ms="5" started-at="2021-04-19T10:49:25 IST" finished-at="2021-04-19T10:49:25 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="m1(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.RegisterTest@1f28c152]" name="m1" duration-ms="2780" started-at="2021-04-19T10:49:25 IST" data-provider="getUserData" finished-at="2021-04-19T10:49:28 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[pistol]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- m1 -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.RegisterTest@1f28c152]" name="precondition" is-config="true" duration-ms="5" started-at="2021-04-19T10:49:28 IST" finished-at="2021-04-19T10:49:28 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="m1(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.RegisterTest@1f28c152]" name="m1" duration-ms="2950" started-at="2021-04-19T10:49:28 IST" data-provider="getUserData" finished-at="2021-04-19T10:49:31 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[eve.holt@reqres.in]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- m1 -->
        <test-method status="PASS" signature="precondition()[pri:0, instance:Assignment.ReqResTesting.RegisterTest@1f28c152]" name="precondition" is-config="true" duration-ms="0" started-at="2021-04-19T10:49:31 IST" finished-at="2021-04-19T10:49:31 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method status="PASS" signature="m1(java.lang.String, java.lang.String)[pri:0, instance:Assignment.ReqResTesting.RegisterTest@1f28c152]" name="m1" duration-ms="2825" started-at="2021-04-19T10:49:31 IST" data-provider="getUserData" finished-at="2021-04-19T10:49:34 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[eve.holt@reqres.in]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[pistol]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- m1 -->
      </class> <!-- Assignment.ReqResTesting.RegisterTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
